packages/core/src/components/popover/popover.tsx

```diff
@@ -252,7 +247,8 @@ export class Popover<
         const { disabled, content, placement, position = "auto", positioningStrategy } = this.props;
         const { isOpen } = this.state;
 
-        const isContentEmpty = content == null || (typeof content === "string" && content.trim() === "");
+        const isContentEmpty =
```

Comment:
this change means the empty check will be hit if the content is `false` or `[]`, which seems like a good change. previously I imagine we tried to render the popover but it ended up not really displaying anything. I can't imagine anyone would want the current behavior.